pkg/vm/engine/disttae/table.go

```diff
@@ -30,6 +30,37 @@ import (
 
 var _ engine.Relation = new(table)
 
+func fixColumnName(tbl *table, expr *plan.Expr) {
+	switch exprImpl := expr.Expr.(type) {
+	case *plan.Expr_F:
+		for _, arg := range exprImpl.F.Args {
+			fixColumnName(tbl, arg)
+		}
+	case *plan.Expr_Col:
+		exprImpl.Col.Name = tbl.tableDef.Cols[exprImpl.Col.ColPos].Name
```

Comment:
Is expr a trimmed expression? Do you have the right name for the mapping like that?

Reply:
The data given by plan should be complete and should not come in other modules to populate the data.